processing/app/database.py

```diff
@@ -0,0 +1,18 @@
+from sqlmodel import SQLModel, create_engine
+
+postgres_user="user"
+postgres_password="password"
+postgres_host="db"
+postgres_db="survey_db"
+postgres_port="5432"
+
+postgres_url = f"postgresql+psycopg://{postgres_user}:{postgres_password}@{postgres_host}:{postgres_port}/{postgres_db}"
+
+engine = create_engine(postgres_url, echo=True, future=True)
```

Comment:
Similar to an other [comment in a different PR](https://github.com/opendexcom/formul.ai/pull/26#discussion_r2069277248) this should be injected via FastAPI dependency.